Documentation/Blazorise.Docs.Compiler/ApiDocsGenerator/README.md

```diff
@@ -0,0 +1,98 @@
+﻿# ApiDocsGenerator
+
+## Which Component Does It Pick Up?
+
+Either one of these cases:
+
+- The component is a descendant of `Blazorise.BaseComponent`.
+- The component is a descendant of `ComponentBase`.
+- The component implements `IComponent`.
+
+### Problem with Razor Files
+
+- It doesn't pick up the parameters from Razor files.
+- For inheritance to be "read," it needs to be explicitly specified inside the `.cs` file:
+  - This is especially true for `ComponentBase`, which is inherited by default when creating a `.razor` file.
+    - You need to explicitly declare `SomeComponent : ComponentBase`.
+
+This limitation can be overcome by scanning through the generated files (`SomeComponent.razor.cs`).
+
+## System.Runtime
+
+Retrieving XML comments from the `System.Runtime` assembly can be problematic, particularly for the `IDisposable` interface.
```

Comment:
Have you tried this library to read XML comments? Maybe they have already solved this kind of problems?

https://github.com/loxsmoke/DocXml

Reply:
The issue I am facing with `System.Runtime` is that it doesn't appear to the analyzer that `IDispose` is from `System.Runtime`. Somehow, it's blind to this assembly, so it "doesn't know which XML to load".